CONTRIBUTING.md

```diff
@@ -47,6 +47,8 @@ Any breaking changes to a chart (backwards incompatible) require:
 
 ### New Application Versions
 
+Helm charts are intended to be created for all relevant releases (Argo CD, Workflows, Rollouts, Events). Associated dependencies, such as Redis, will use the version recommended by the associated release.
```

Comment:
What does that mean "for all relevant releases"? Right now we cannot release a new chart version for a patch happening on the non-mainline version.

Reply:
@mkilchhofer or @jmeridth, could you suggest a revision to increase the precision here? To be honest I was being somewhat vague in summarizing thoughts that JM shared offline

Reply:
s/for all relevant releases/for all non-patched releases

Reply:
applied, and went back now to push not-from-mobile for DCO